src/index.js

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+
+import styled from 'styled-components';
+import CssBaseline from '@material-ui/core/CssBaseline'
+
+import Header from "./components/Header";
+
+
+const App = () => (
+  <Root>
+    <CssBaseline/>
+    <Header/>
+    <p> Page content here </p>
+  </Root>
+);
+
+const Root = styled.div`
+  background: lightgrey;
+`;
```

Comment:
Have you heard of CSS Modules? We can either use this approach or stick to styled-components.
I'm okay with both, so it's up to you.

Reply:
Let's proceed with this approach